tests/testthat/listing.qmd

```diff
@@ -0,0 +1,6 @@
+---
+title: "Listing"
```

Comment:
I'd prefer the contents of this file to be created from within the test. This way when reading the test, we can see what listing.md is without having to go open it. Something like (I might have gotten indentation wrong):

```r
listing_lines <- c(
    "---",
    'title: "Listing"',
    "listing:",
    '  - contents: ["subdir/*.qmd"]',
    "    type: grid",
    "---",
)
brio::write_lines(
    listing_lines, 
    file.path(parent_dir, project_dir, "listing.qmd")
)
```

Reply:
Thanks for the code suggestion. I didn't know how to proceed for this.